app/controllers/ontologies_controller.rb

```diff
@@ -320,6 +321,28 @@ def submit_success
     render 'submit_success'
   end
 
+  # GET /ontologies/:acronym/:id
+  def redirect
+    return not_found unless params[:acronym] && params[:id]
+
+    request_accept_header = request.env["HTTP_ACCEPT"].split(",")[0]
+    type, resource_id  = find_type_by_id(params[:id], params[:acronym])
+
+    return not_found if resource_id.nil?
```

Comment:
if resource_id is nil, redirect to the summary page of the ontology. 
if the ontology does not exist return ontology_not_found

Reply:
done in https://github.com/ontoportal-lirmm/bioportal_web_ui/pull/568/commits/e34abdede9fbe849fb603b6e0d907df8228c0dae